buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/converter.py

```diff
@@ -0,0 +1,179 @@
+import dateutil
+from pathlib import Path
+from datetime import datetime
+
+import numpy as np
+
+from neuroconv import NWBConverter
+
+from buzsaki_lab_to_nwb.huszar_hippocampus_dynamics.behaviorinterface import HuzsarBehaviorInterface
+
+
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import scipy
+
+from neuroconv.datainterfaces.ecephys.basesortingextractorinterface import BaseSortingExtractorInterface
+from neuroconv.tools import get_package
+from neuroconv.utils import FilePathType
+
+
+class CellExplorerSortingInterface(BaseSortingExtractorInterface):
```

Comment:
Can you move this to a separate file for better organization?

Reply:
Also see Petersens e-mail; we should just need to set the upstream sorting extractor to allow `session.mat` instead of `sessionInfo`

Reply:
Yes, I will. I will probably just roll this all the way out to Neuroconv. There is no `sampling_rate` in `session.mat` So, I will move stuff but that specific route can't be done.